examples/connext_dds/network_capture/01_shared_memory_and_udp/cs/NetworkCapture.idl

```diff
@@ -9,6 +9,10 @@
  * not be liable for any incidental or consequential damages arising out of the
  * use or inability to use the software.
  */
-struct NetworkCapture {
-    string<128> msg;
+module NetworkCaptureExample {
+
+    struct HelloWorld {
+        string<128> msg;
+    };
+
 };
```

Comment:
I wrapped the type in a module, as recommended by reviewer.
However, I left the rest of the type intact, so that the type is compatible with the rest of the APIs.